src/player_hardcoded_effects.cpp

```diff
@@ -1336,14 +1336,15 @@ void player::hardcoded_effects( effect &it )
                         // 10 minute cyber-snooze
                         it.mod_duration( 100 );
                     }
-                } else {
+                }
+            } else {
+                if( dur <= 1 ) {
+                    add_msg_if_player( m_warning, _( "Your alarm clock rings, but you do not hear it." ) );
```

Comment:
@kevingranade @Qrox I think a simple change to the message would be fine.

E.g. `You check the time - looks like you slept through your alarm.`